### PR TITLE
Use "eslint" as the default linter and code formatter

### PR DESCRIPTION
**Pull request by dgaubert**
- Follow  standard as base configuration with a couple of tweaks
  - 4 spaces
  - semicolons always

### CODE REVIEW

File: lib/models/query_metadata.js [hunk 1]
```diff
@@ -19,49 +19,47 @@ class QueryMetadata {
      * Returns the calculated X-Cache-Channel for all of the tables.
      * @returns {String}
      */
-    getCacheChannel(skipNotUpdatedAtTables) {
+    getCacheChannel (skipNotUpdatedAtTables) {
         const groupedTables = this.getTables(skipNotUpdatedAtTables).reduce((grouped, table) => {
-            if (!grouped.hasOwnProperty(table.dbname)) {
+            if (!Object.prototype.hasOwnProperty.call(grouped, table.dbname)) {
```
Comment (Algunenano) [inline lib/models/query_metadata.js]: What's with these changes (a.hasOwnProperty -> Object.prototype...)? I don't think it's more readable now than before.

Comment (dgaubert) [reply lib/models/query_metadata.js]: https://eslint.org/docs/rules/no-prototype-builtins

Comment (dgaubert) [reply lib/models/query_metadata.js]: Worse readability but safer.... :(

Comment (Algunenano) [reply lib/models/query_metadata.js]: :ok_hand:

**Comment by dgaubert** [comment]
> I'm not sure about the space between function name and parameters

It's a matter to get used to it. 🤷‍♂ 

I don't agree with many rules (2 spaces or semicolon) but we need to standardize the way we write javascript in the backend. I'd even like to reach a consensus with frontend but I don't feel strong enough to move this forward.

In any case, you can create a PR with the changes in `.eslintrc.js` file with the customizations that you consider. We can reach an agreement. Please, have in mind I would like to be as close to the community standards as possible. 

https://www.npmtrends.com/eslint-config-standard-vs-eslint-config-airbnb-base-vs-eslint-config-google

I've chosen `eslint-config-standard` despite is the second most used. First, I've tried `eslint-config-airbnb-base` but it has some rules that really disturbed me. I'm still open to it if you want to give a try and you consider it's worth to use it as our new standard.

**Comment by Algunenano** [comment]
> I don't agree with many rules (2 spaces or semicolon) but we need to standardize the way we write javascript in the backend.

I can get used to it. Please set a standard that isn't crazy (like you just did) and we can apply it to all nodejs projects.  You have at least 66+% of the votes.

**Pull request merged**